vllm/transformers_utils/configs/dbrx.py

```diff
@@ -0,0 +1,275 @@
+# coding=utf-8
+# Copied from
+# https://huggingface.co/databricks/dbrx-base/blob/main/configuration_dbrx.py
+"""Dbrx configuration."""
+
+from typing import Any, Optional
+
+from transformers.configuration_utils import PretrainedConfig
+from transformers.utils import logging
+
+logger = logging.get_logger(__name__)
+
+DBRX_PRETRAINED_CONFIG_ARCHIVE_MAP = {}
```

Comment:
Do we need this?

Reply:
Was following the same format as HF `configuration_dbrx.py` (https://huggingface.co/databricks/dbrx-base/blob/4a0342461bb367b6f9880733d3c1160d640f2942/configuration_dbrx.py#L9)

Reply:
Got it. Thanks for the explanation.